src/pages/user/authority/OrcidModal.tsx

```diff
@@ -28,7 +28,7 @@ const OrcidModal: FC<OrcidModalProps> = ({ closeModal }) => {
       ariaLabelledBy="orcid-modal"
       openModal={true}
       onClose={closeModal}
-      maxWidth="lg"
+      maxWidth="md"
```

Comment:
<img width="1221" alt="Skjermbilde 2020-06-12 kl  12 54 37" src="https://user-images.githubusercontent.com/6595982/84495694-e20b5500-acab-11ea-8ff8-8287919507ca.png">

Synes denne er alt for stor.
```suggestion
      maxWidth="sm"
```